src/xrpld/overlay/detail/PeerImp.cpp

```diff
@@ -2936,7 +3078,9 @@ getPeerWithLedger(
 void
 PeerImp::sendLedgerBase(
     std::shared_ptr<Ledger const> const& ledger,
-    protocol::TMLedgerData& ledgerData)
+    protocol::TMLedgerData& ledgerData,
+    std::map<std::shared_ptr<Peer>, std::set<std::optional<uint64_t>>> const&
```

Comment:
`std::map<std::shared_ptr<Peer>, std::set<std::optional<uint64_t>>>` is mentioned in four places. It would be more readable to define an alias:
`using PeerCookieMap = std::map<std::shared_ptr<Peer>, std::set<std::optional<uint64_t>>>;`

Reply:
Good suggestion! Fixed.